scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/htssop.yaml

```diff
@@ -115,6 +115,36 @@ HTSSOP-20-1EP_4.4x6.5mm_P0.65mm_EP3.4x6.5mm_Mask2.4x3.7mm:
   EP_mask_y: 3.7
   EP_num_paste_pads: [1, 2]
 
+HTSSOP-20-1EP_4.4x6.5mm_P0.65mm_EP2.85x4mm:
+  size_source: 'https://pdfserv.maximintegrated.com/package_dwgs/21-0108.PDF'
```

Comment:
Please append `U20E-1` package code to URL string.

Reply:
Done, same as above?

Reply:
Github was having some issues so some comments got posted twice, sry.